problems/lib/problem1.rb

```diff
@@ -0,0 +1,7 @@
+class Problem1
+  def sum_even_numbers(array)
+    sum = 0

```

Comment:
Oh but ruby is so much beautiful can you do this on one line? 

_Hint, checkout `select`, and `[].sum == 0)_